### DISCUSSION TITLE
Ongoing: DNS Check in Progress

### DISCUSSION THREAD

**Discussion by JonnyTech** [community/community#57680] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Github pages shows `DNS Check in Progress` on my custom domain but it is served correctly and even redirects http:// to https:// - is this a bug?

**Comment by JonnyTech** [comment]
I see that the `more information needed` label has been applied to this discussion, but no request has been made for what information is required...

Repository: https://github.com/wimllc/devsite
Custom domain: https://dev.wim.ae/
Added, removed, added... both CNAME and TXT records

**Reply by neutrinomuon** [reply]
Hi @JonnyTech ,

I hope you're doing well. I've entered in your repository and website and found no problem. Maybe it was a momentary error?

**Reply by JonnyTech** [reply]
Thanks Jean, much appreciated, I guess that I shall just ignore the message and hope that GitHub clears it soon..

**Comment by JonnyTech** [comment]
Maybe I should keep this issue open until it clears because it may be an actual bug...?

**Comment by RFPanda** [comment]
The problem is still relevant. What can be done?
DNS_PROBE_FINISHED_NXDOMAIN

**Comment by MrMrch** [comment]
Im also having issues. Still DNS in progress after multiple hours, most people can’t access the link and https can’t be setup. 
milk wait the morning and see if it’s propagated

**Reply by city41** [reply]
I am as well. My site has been under "DNS check in progress" for 24 hours now.

**Reply by BenLeeder** [reply]
Same here did you find a solution?

**Comment by munizigor** [comment]
Same problem here! Almost a week with "DNS check in progress"

**Reply by city41** [reply]
I plan to move to vercel. This feels like a part of GitHub that has been abandoned, sadly.

**Comment by resurs365** [comment]
Same here. "DNS Check in Progress" forever.

**Comment by FidelusAleksander** [comment]
Same here on https://github.com/FidelusAleksander/githubcertified

**Comment by SarahNv** [comment]
Was anyone able to figure out the issue?

**Comment by spsrinivas** [comment]
Same here. It's been there for months now.

**Comment by ju-mc** [comment]
Also having this problem

**Comment by JonnyTech** [comment, marked as answer]
I realised that out of several domains that I had linked, one was pointing to an incorrect organisation. Once I rectified that then the notifications ceased. Check all DNS records of linked accounts, the message, although vague and rather misleading, is definitely there for a reason.

**Reply by LiteBrite82** [reply]
**Moderators note: See also** https://github.com/orgs/community/discussions/57680#discussioncomment-8521799
Thanks to @elawad and @shafyq

**Comment by elawad** [comment]
To resolve: Click Remove, add the domain again, click Save.
Was verified in seconds afterwards.

**Reply by rajanbajaj** [reply]
This helped. Thanks!

**Reply by ghost** [reply]
> To resolve: Click Remove, add the domain again, click Save. Was verified in seconds afterwards.

This one  + adding A and AAAA records for www and IPV6 solved the issue for me
Kudos

**Comment by pplmx** [comment]
The same issue since recent weeks.

**Reply by k4r4b3y** [reply]
try this: https://github.com/orgs/community/discussions/57680#discussioncomment-8521799

**Comment by sbarbett** [comment]
I'd like to add - If you are using Cloudflare for DNS and experiencing issues, disable the proxy service on Cloudflare.
